### ISSUE TITLE
System Color Problem

### ISSUE THREAD

**Issue by lainz** [bgrabitmap/bgrabitmap#40]
Hi, this issue was created in bgracontrols, but it belongs here
https://github.com/bgrabitmap/bgracontrols/issues/26

**Comment by lainz** [comment]
I will fix it in BGRAControls, as proposed by the reporter so it works properly now.

**Comment by circular17** [comment]
Hi Sorry I realized I did not reply to this issue. I try to remember why I was not fond of changing this function as it has been a long time like that. I think that maybe I want to avoid LCL dependency. Though I made some changes to make it independent when necessary from LCL and wrote some ColorToRGB equivalent depending on the backend. Also I was like obsessed with optimizations not wanting to add just one test to check if it was an RGB color or a system color. But in fact I realized doing one "if" is very fast, so I guess my reluctance does not make sense.
